src/components/Query.ts

```diff
@@ -178,8 +178,16 @@ export class APIRequest {
       )}&q=${encodedQuery}`
       data = (await axios({ method: 'GET', url, cancelToken: source.token })).data
     } catch (err) {
-      const message = err.response?.data?.message || err.toString()
```

Comment:
this could just be throw new Error(message.toString()) and have the same effect without the type checking

Reply:
err.toString() will always returns an error, but not necessary a message from server. The issue we are having is connection error not displayed correctly. which results to Error: Error: connection refused

Reply:
toString returns a string rather than an instance of error